### DISCUSSION TITLE
Show objects decription for Oracle tables

### DISCUSSION THREAD

**Discussion by plustag** [dbeaver/dbeaver#17259] [Q&A]
### Description

I checked "Show objects description" in Connection view.
but does not display description in table list in tree.
However it started to show after View Table(F4)

### Steps to reproduce

oracle 19c

### DBeaver Version

22.1.2

### Operating System

windows 10

**Comment by LonwoLonwo** [comment]
Hello @plustag 

Comments metadata is too expansive. For this reason, we read comments only when we read table metadata. 

You can open (double-click on the folder) "Tables" in the Database Navigator tree, and all comments will be read for all tables.

![2022-08-04 10_26_38-DBeaver Enterprise 22 1 3 - HR](https://user-images.githubusercontent.com/45152336/182789312-b769c173-ceab-4786-a1ad-59981421f888.png)

**Reply by plustag** [reply]
Thanks your reply. I got a new tips. :)
